Pod/Core/TypeConversion.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TypeConversion.swift
+//  Pods
+//
+//  Created by Sam Dean on 4/4/17.
+//
+//
+
+import Foundation
+
+
+public protocol FromStringable {
```

Comment:
Ah, also we probably want to convert case insensitive true/false to Bool - and init?(_:) on Bool only converts lowercase :(